.github/workflows/ci-core.yml

```diff
@@ -120,6 +120,26 @@ jobs:
           certManager:
             enabled: ${{ matrix.enableCertManager }}
             generateCA: true
+        extraObjects:
+          - apiVersion: v1
+            kind: Secret
+            metadata:
+              name: aws-credentials
+              namespace: keda
+            type: Opaque
+            data:
+              AWS_ACCESS_KEY_ID: RkFLRUFXU0FDQ0VTU0tFWUlE
+              AWS_SECRET_ACCESS_KEY: RkFLRUFXU1NFQ1JFVEFDQ0VTU0tFWQ==
```

Comment:
I'd change it to something different than a Secret. Having a Secret in an example will encourage users to do this and having plain secrets in a section what is human readable is not nice.

Maybe something like we use in argo-helm would fit better:
https://github.com/argoproj/argo-helm/blob/5873a1bc331e37c9500e4337d2ad2deca2fd8c80/charts/argo-cd/values.yaml#L535-L560

Or if you want to do an example with aws-creds I'd use at least 
- `kind: SealedSecret` (https://github.com/bitnami-labs/sealed-secrets)
- `kind: ExternalSecret` (https://external-secrets.io/)

Reply:
Gotcha